Add get user profile info endpoint

marynakernychna opened the pull request:
> Use Postman to check

marynakernychna committed:
Add get user profile info endpoint

 > Use Postman to check

pashunskyi commented on API/Controllers/UserController.cs:
```diff
@@ -0,0 +1,26 @@
+            _userService = userService;
+        }
+
+        [HttpGet("get-user-info")]
```
Are you sure this is the correct API?

marynakernychna committed:
Changing the logic of getting the user id

> Rename the path
> Add jwt authentication
> Delete unnecessary fields from user dto

marynakernychna committed:
Add a method to get the user id

> Add a method to get the user id
> Add error message
